dockerfiles/base/scripts/base/commands/cmd_action.sh

```diff
@@ -10,9 +10,31 @@
 #
 
 cmd_action() {
-  debug $FUNCNAME
+  docker_run -it ${UTILITY_IMAGE_CHEACTION} "$@"
+}
+
+pre_cmd_action() {
```

Comment:
I thought about that but had problems with delegation.  The output was not controllable and some of the explanations did not make sense.  I was worried about our if syncness like you said but could not figure out a clean way to present the information.